sensor/tests/connection/k8sreconciliation/reconcile_test.go

```diff
@@ -31,27 +34,59 @@ func Test_SensorReconcilesKubernetesEvents(t *testing.T) {
 	c, err := helper.NewContext(t)
 	require.NoError(t, err)
 
+	// Timeline of the events in this test:
+	// 1) Create deployment Nginx1
+	// 2) Create deployment Nginx2
+	// 3) Create NetworkPolicy block-all-egress
+	// 4) gRPC Connection interrupted
+	// 5) Delete deployment Nginx2
+	// 6) Create deployment Nginx3
+	// 7) gRPC Connection re-established
+	// 8) Sensor transmits current state with SYNC event type:
+	//  Deployment 		Nginx1
+	//  Deployment 		Nginx3
+	//  NetworkPolicy 	block-all-egress
+	//
+	// Using a NetworkPolicy here will make sure that no deployments that were removed while the connection
+	// was down will be used for
```

Comment:
Will be used for what?

Reply:
Whoops, small attention span to blame. Fixed